scripts/lib/install

```diff
@@ -586,13 +586,21 @@ EOF
 fi
 
 if has_class "zulip::app_frontend_base"; then
+    # Frontend deploys use /home/zulip/deployments; without this, the
+    # install directory is also only readable by root.
```

Comment:
This comment only makes sense when it's got a chown or something after it.  The chown is still down at line 706, so the comment doesn't make sense.